Lib/test/test_ipaddress.py

```diff
@@ -476,6 +476,12 @@ class InterfaceTestCase_v4(BaseTestCase, NetmaskTestMixin_v4):
 class NetworkTestCase_v4(BaseTestCase, NetmaskTestMixin_v4):
     factory = ipaddress.IPv4Network
 
+    def test_no_mask(self):
```

Comment:
I suggest move this equal test to `NetmaskTestMixin_v4.test_valid_netmask`. I could also be applied to ip interface objects then.

Reply:
Moved, thanks.